window-manipulate/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "applications": {
+    "gecko": {
+      "id": "window-manipulate@mozilla.org",
+      "strict_min_version": "47.0a1"
+    }
+  },
+  "browser_action": {
+    "browser_style": true,
+    "default_title": "Window manipulate",
```

Comment:
Could we use "Window manipulator" or something like that instead? Here, and also in the title of the popup, and the "name" field.